src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/MarkerVisual/MarkerVisualSpatialLocalizer.cs

```diff
@@ -6,6 +6,7 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using UnityEditor;
 using UnityEngine;
```

Comment:
Is this needed? nit: please revert this file if no changes occurred.

Reply:
I think these were remnants from an intermediate change, will revert them.